source/renderer/app/components/widgets/forms/TinySwitch.scss

```diff
@@ -0,0 +1,30 @@
+.root {
+  flex-direction: row !important;
+
+  :global {
+    .SimpleSwitch_switch {
+
+      &.SimpleSwitch_checked,
+      &:not(.SimpleSwitch_checked) {
```

Comment:
@daniloprates I am not sure if I follow this line - looks like you are targeting both elements with `. SimpleSwitch_checked` class and all which are not of that class?

Reply:
Before using `:global` this was the only way to override the component rule. Forgot to remove it afterwards.